nut/DOCS.md

```diff
@@ -194,13 +176,55 @@ When this option is set to `true` on a UPS shutdown command, the host system
 will be shutdown. When set to `false` only the add-on will be stopped. This is to
 allow testing without impact to the system.
 
+### Option: `log_level`
+
+This option controls the level of log output by the add-on's scripts, which
+might be useful when troubleshooting the add-on itself. Possible values are:
+
+- `trace`: Show every detail, like all called internal functions.
+- `debug`: Shows detailed debug information.
+- `info`: Normal (usually) interesting events.
+- `warning`: Exceptional occurrences that are not errors.
+- `error`: Runtime errors that do not require immediate action.
+- `fatal`: Something went terribly wrong. Add-on becomes unusable.
+
+Please note that each level automatically includes log messages from a
+more severe level, e.g., `debug` also shows `info` messages. By default,
+the `log_level` is set to `info`, which is the recommended setting unless
+you are troubleshooting.
+
+### Option: `driver_debug`
+
+This option controls the level of log output by NUT drivers. Set this to an
+integer between 0 and 6, where 0 logs only errors and 6 logs all debug messages.
+This is equivalent to setting `debug_min = <value>` in the `config` sub-option
+of every `device`.
+
+### Option: `LIBUSB_DEBUG`
+
+This option controls the level of log output by libusb. Set this to an integer
+between 0 and 4, where 0 logs nothing and 4 logs all debug messages. This is
+equivalent to setting `LIBUSB_DEBUG = <value>` in the `config` sub-option of
+every `device`.
```

Comment:
Please use the existing log level to hook into instead of adding more options.

Thanks! 👍

Reply:
Done.